underfs/hdfs/src/main/java/alluxio/underfs/hdfs/activesync/SupportedHdfsActiveSyncProvider.java

```diff
@@ -132,7 +132,8 @@ public SupportedHdfsActiveSyncProvider(URI uri, org.apache.hadoop.conf.Configura
    * unsynced syncpoints.
    */
   private void initNextWindow() {
-    for (String syncPoint : mActivity.keySet()) {
+    for (Map.Entry<String, Integer> activity : mActivity.entrySet()) {
+      String syncPoint = activity.getKey();
       mActivity.put(syncPoint, mActivity.get(syncPoint) / 10);
```

Comment:
i think you meant to change this line as well? to use activity.getValue()?

Reply:
Good catch!